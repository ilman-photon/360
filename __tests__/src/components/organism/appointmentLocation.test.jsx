import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import AppointmentLocation from "../../../../src/components/organisms/ScheduleAppointment/appointmentLocation";

describe("App", () => {
    let container;
    beforeEach(() => {
      container = render(<AppointmentLocation />);
    });
  
    it("AppointmentLocation render", () => {
        
    });
    
  });
  
