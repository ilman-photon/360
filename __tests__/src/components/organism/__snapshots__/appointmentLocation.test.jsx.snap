// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App AppointmentLocation render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiBox-root css-1qm1lh"
      >
        <div
          class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root card css-1v5z4dq-MuiPaper-root-MuiCard-root"
        >
          <div
            class="MuiCardHeader-root cardHeader css-1ele1k2-MuiCardHeader-root"
          >
            <div
              class="MuiCardHeader-content css-1qbkelo-MuiCardHeader-content"
            >
              <span
                class="MuiTypography-root MuiTypography-h5 MuiCardHeader-title css-1qvr50w-MuiTypography-root"
              >
                <div>
                  <span
                    style="margin-right: 8px; display: flex; align-items: center;"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                      data-testid="LocationOnIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M12 2C8.13 2 5 5.13 5 9c0 5.25 7 13 7 13s7-7.75 7-13c0-3.87-3.13-7-7-7zm0 9.5c-1.38 0-2.5-1.12-2.5-2.5s1.12-2.5 2.5-2.5 2.5 1.12 2.5 2.5-1.12 2.5-2.5 2.5z"
                      />
                    </svg>
                  </span>
                  <div
                    class="title"
                    style="font-weight: 700;"
                  >
                    location
                  </div>
                </div>
              </span>
            </div>
            <div
              class="MuiCardHeader-action css-sgoict-MuiCardHeader-action"
            >
              <button
                class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root  css-1e6y48t-MuiButtonBase-root-MuiButton-root"
                tabindex="0"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1xhbeow-MuiSvgIcon-root"
                  data-testid="EditOutlinedIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="m14.06 9.02.92.92L5.92 19H5v-.92l9.06-9.06M17.66 3c-.25 0-.51.1-.7.29l-1.83 1.83 3.75 3.75 1.83-1.83c.39-.39.39-1.02 0-1.41l-2.34-2.34c-.2-.2-.45-.29-.71-.29zm-3.6 3.19L3 17.25V21h3.75L17.81 9.94l-3.75-3.75z"
                  />
                </svg>
                <div
                  style="margin-left: 4px; color: rgb(0, 130, 148); text-decoration-line: underline; text-transform: capitalize; font-size: 18px;"
                  type="link"
                >
                  Edit
                </div>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </div>
          </div>
          <div
            class="MuiCardContent-root css-vp7ejv-MuiCardContent-root"
          >
            <div
              class="css-1p5q5e5-MuiStack-root"
            >
              <div
                class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
              >
                <div
                  class="MuiGrid-root css-1wgc5ak-MuiGrid-root"
                >
                  <span
                    style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
                  >
                    <span
                      style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                    >
                      <img
                        alt=""
                        aria-hidden="true"
                        src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27105%27%20height=%27105%27/%3e"
                        style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                      />
                    </span>
                    <img
                      alt="profile"
                      data-nimg="intrinsic"
                      decoding="async"
                      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                      style="border-radius: 50%; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                    />
                    <noscript />
                  </span>
                </div>
                <div
                  class="MuiGrid-root css-1dhf0bi-MuiGrid-root"
                >
                  <h4
                    aria-label="Myself"
                    class="MuiTypography-root MuiTypography-h4 css-5lbw0b-MuiTypography-root"
                    style="color: rgb(0, 59, 74); font-size: 18px; font-weight: 700;"
                  />
                  <span
                    aria-label="Myself"
                    class="MuiTypography-root MuiTypography-regularBold css-37jdci-MuiTypography-root"
                    style="color: rgb(0, 59, 74); font-size: 16px;"
                  >
                    <br />
                  </span>
                  <h4
                    aria-label="Myself"
                    class="MuiTypography-root MuiTypography-h4 css-5lbw0b-MuiTypography-root"
                    style="color: rgb(0, 59, 74); font-size: 16px;"
                  >
                    <a
                      class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1ps4owl-MuiTypography-root-MuiLink-root"
                      style="color: rgb(0, 130, 148); font-size: 18px;"
                    />
                  </h4>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiBox-root css-1qm1lh"
    >
      <div
        class="MuiPaper-root MuiPaper-outlined MuiPaper-rounded MuiCard-root card css-1v5z4dq-MuiPaper-root-MuiCard-root"
      >
        <div
          class="MuiCardHeader-root cardHeader css-1ele1k2-MuiCardHeader-root"
        >
          <div
            class="MuiCardHeader-content css-1qbkelo-MuiCardHeader-content"
          >
            <span
              class="MuiTypography-root MuiTypography-h5 MuiCardHeader-title css-1qvr50w-MuiTypography-root"
            >
              <div>
                <span
                  style="margin-right: 8px; display: flex; align-items: center;"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="LocationOnIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M12 2C8.13 2 5 5.13 5 9c0 5.25 7 13 7 13s7-7.75 7-13c0-3.87-3.13-7-7-7zm0 9.5c-1.38 0-2.5-1.12-2.5-2.5s1.12-2.5 2.5-2.5 2.5 1.12 2.5 2.5-1.12 2.5-2.5 2.5z"
                    />
                  </svg>
                </span>
                <div
                  class="title"
                  style="font-weight: 700;"
                >
                  location
                </div>
              </div>
            </span>
          </div>
          <div
            class="MuiCardHeader-action css-sgoict-MuiCardHeader-action"
          >
            <button
              class="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root  css-1e6y48t-MuiButtonBase-root-MuiButton-root"
              tabindex="0"
              type="button"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1xhbeow-MuiSvgIcon-root"
                data-testid="EditOutlinedIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="m14.06 9.02.92.92L5.92 19H5v-.92l9.06-9.06M17.66 3c-.25 0-.51.1-.7.29l-1.83 1.83 3.75 3.75 1.83-1.83c.39-.39.39-1.02 0-1.41l-2.34-2.34c-.2-.2-.45-.29-.71-.29zm-3.6 3.19L3 17.25V21h3.75L17.81 9.94l-3.75-3.75z"
                />
              </svg>
              <div
                style="margin-left: 4px; color: rgb(0, 130, 148); text-decoration-line: underline; text-transform: capitalize; font-size: 18px;"
                type="link"
              >
                Edit
              </div>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
        <div
          class="MuiCardContent-root css-vp7ejv-MuiCardContent-root"
        >
          <div
            class="css-1p5q5e5-MuiStack-root"
          >
            <div
              class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
            >
              <div
                class="MuiGrid-root css-1wgc5ak-MuiGrid-root"
              >
                <span
                  style="box-sizing: border-box; display: inline-block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: relative; max-width: 100%;"
                >
                  <span
                    style="box-sizing: border-box; display: block; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; max-width: 100%;"
                  >
                    <img
                      alt=""
                      aria-hidden="true"
                      src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27105%27%20height=%27105%27/%3e"
                      style="display: block; max-width: 100%; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px;"
                    />
                  </span>
                  <img
                    alt="profile"
                    data-nimg="intrinsic"
                    decoding="async"
                    src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                    style="border-radius: 50%; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%;"
                  />
                  <noscript />
                </span>
              </div>
              <div
                class="MuiGrid-root css-1dhf0bi-MuiGrid-root"
              >
                <h4
                  aria-label="Myself"
                  class="MuiTypography-root MuiTypography-h4 css-5lbw0b-MuiTypography-root"
                  style="color: rgb(0, 59, 74); font-size: 18px; font-weight: 700;"
                />
                <span
                  aria-label="Myself"
                  class="MuiTypography-root MuiTypography-regularBold css-37jdci-MuiTypography-root"
                  style="color: rgb(0, 59, 74); font-size: 16px;"
                >
                  <br />
                </span>
                <h4
                  aria-label="Myself"
                  class="MuiTypography-root MuiTypography-h4 css-5lbw0b-MuiTypography-root"
                  style="color: rgb(0, 59, 74); font-size: 16px;"
                >
                  <a
                    class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1ps4owl-MuiTypography-root-MuiLink-root"
                    style="color: rgb(0, 130, 148); font-size: 18px;"
                  />
                </h4>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
